### ISSUE TITLE
System requirements for t5-11b model

### ISSUE THREAD

**Issue by exelents** [microsoft/DeepSpeed#655]
In descriptions of DeepSpeed I see that it allows training up to 13 bilions parameters model on a single GPU. What is the memory requirements were for this? I see information about a single V100 - was it v100 with 16 GB memory or 32 GB?
And a final question: is there a way to run google's t5-11b model on a GPU with 24 GB of memory (rtx3090) ? fp16 mode and cpu_offload turned on. Here is my config:
https://github.com/exelents/try_t5_qa/blob/main/ds_config.json

**Comment by tjruwase** [comment]
@exelents Thanks for using DeepSpeed. 

13B parameter requires 32GB V100. Have you seen this [tutorial](https://www.deepspeed.ai/tutorials/zero-offload/)?

The config looks fine for running in fp16 and cpu_offload. Can you please try it and share your experience?

**Comment by exelents** [comment]
> 13B parameter requires 32GB V100

That's a pity. I tried to finetune T5 with 11B paramenters on 24GB RTX3090. Seems, it's not enough, because I got an Cuda OOM error.

**Comment by tjruwase** [comment]
Yes, unfortunately, for a 11B model, just the model parameters will consume 22GB and there will nothing left for activations or gradients on 24GB GPU.
